ruby_v2/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ruby:3.2.1-alpine3.16
+
+ENV APP_ROOT=/ruby_v2
+
```

Comment:
It would be nice to test with and without `ENV RUBYOPT="--yjit"`
What do you think?

Reply:
I did, on my machine I had the following results:
No --yjit
```
/ruby # /scripts/stats ruby
File: ruby-1676922947.csv
Processed- 1000000
sec- 0; TH- 71913
sec- 1; TH- 76335
sec- 2; TH- 76327
sec- 3; TH- 76390
sec- 4; TH- 76167
sec- 5; TH- 76213
sec- 6; TH- 76217
sec- 7; TH- 76136
sec- 8; TH- 76215
sec- 9; TH- 76169
sec- 10; TH- 76116
sec- 11; TH- 76206
sec- 12; TH- 76113
sec- 13; TH- 13483

Avg items/s- 76217
Min items/s- 76113
Max items/s- 76390
Signature- 24b760abba866d98a760b0cac44543b2
Expected - 24b760abba866d98a760b0cac44543b2
```

with --yjit
```
/ruby # /scripts/stats ruby
File: ruby-1676923083.csv
Processed- 1000000
sec- 0; TH- 9533
sec- 1; TH- 89508
sec- 2; TH- 89694
sec- 3; TH- 89497
sec- 4; TH- 89337
sec- 5; TH- 84695
sec- 6; TH- 88181
sec- 7; TH- 88733
sec- 8; TH- 89268
sec- 9; TH- 89100
sec- 10; TH- 89303
sec- 11; TH- 89148
sec- 12; TH- 14003

Avg items/s- 88769
Min items/s- 84695
Max items/s- 89694
Signature- 24b760abba866d98a760b0cac44543b2
Expected - 24b760abba866d98a760b0cac44543b2
```
Maybe we can add a another runner that passes the flag to ruby command, so we can just
```
./runner-yjit
```
🤔

Reply:
Yeah, it would be good to have such option somewhere
Do we want to rerun all tests someday? ie [RESULTS.md](https://github.com/evkorotkov/RedisCrunchChallenge/blob/master/RESULTS.md)